src/ui.c

```diff
@@ -43,38 +45,44 @@
 #define EX_SIG 128
 #define EX_SIGINT (EX_SIG + SIGINT)
 
-int stdoutfd;
-
 void int_handler();
+int putch(int);
+void putstrat(int, int, char *);
+void movecursto(int, int);
+
+FILE *tty;
+struct termios oldattr;
+int using_alternate_screen;
 
 void
 start_curses()
 {
-	int fd;
+	struct termios newattr;
+	if ((tty = fopen("/dev/tty", "w+")) == NULL) {
```

Comment:
How about a newline between the declarations and the start of the code (the `if`)?